qualification/conftest.py

```diff
@@ -161,13 +167,24 @@ def pytest_generate_tests(metafunc: pytest.Metafunc) -> None:
         metafunc.parametrize("n_antennas", values)
     if "band" in metafunc.fixturenames:
         metafunc.parametrize("band", metafunc.config.getini("bands"))
-    if "n_channels" in metafunc.fixturenames or "narrowband_decimation" in metafunc.fixturenames:
+    if (
+        "n_channels" in metafunc.fixturenames
+        or "narrowband_decimation" in metafunc.fixturenames
+        or "vlbi_decimation" in metafunc.fixturenames
```

Comment:
I don't think we have a *fixture* called vlbi_decimation, just an ini option.

Reply:
Noted. Although, I didn't see a fixture called `narrowband_decimation` either. Thought this was more of a future-proofing/safety addition.

Also, I was thinking around "what would prompt this branch to execute". Either way, will revert.

Reply:
It's not declared with `@pytest.fixture` (because it gets defined here), but it's used in parameter lists of other fixtures like `_cbf_config_and_description`.